rules/game_process.tex

```diff
@@ -75,10 +81,20 @@ \subsection{Robot States}
   \label{fig:robot_states}
 \end{figure}
 
-The referee will announce the transition from the \texttt{initial} to \texttt{ready} state by raising both hands over its head. The referee will wear red gloves \footnote{Teams are encouraged to not rely on the red gloves as they could be removed in a following year.}.
-The referee must stand on the opposite side of the technical area behind the touchline and at the height of the halfway line.
+At the scheduled time, the referee will announce that the transition from the \texttt{initial} to \texttt{visual} will be executed 60 seconds later.
+The referee shall call ``Start in 60 seconds''
+The GameController operator shall start the countdown through the GameController.
+The GameController operator will announce the transition from \texttt{initial} to \texttt{visual} when the head referee is calling ``Start''.
```

Comment:
Here we normally just call the current start and I wouldn't call "Start" here since this assumes that the game really starts and then nothing happens for a random amount of time. So here it would also come in handy if the state is called Standby and the referee could just announce Standby.